cms/tags.yaml

```diff
@@ -6,5 +6,4 @@ rhel7-itb: rhel7-m20220512
 rhel8-itb: cmssw/el8:x86_64
 rhel6: rhel6-m20211005
 rhel7: rhel7-m20220331
-rhel8: rhel8-m-m20220516
-rhel8-m: rhel8-m-m20221009
+rhel8: cmssw/el8:x86_64
```

Comment:
@mmascher , do you really want to go with `cmssw/el8:x86_64` moving target ? Note that `cmssw/el8:x86_64` automatically gets rebuild every time the base image (almalinux:8) is updated or if we have added a new/remove any package.

Reply:
Ah no, sorry, I misunderstood in the ticket. Can we create a tag or something? Also, please wait until Monday before merging, please :)

Reply:
Just read the ticket again. I will use `cmssw/el8:x86_64-d20231016`

Reply:
the latest tag (as mentioned in the Jira ticket) is `cmssw/el8:x86_64-d20231016`  which was built on 16th OCT